src/js/components/Beacon/BeaconErrorMessage.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { Alert } from 'antd';
+
+const BeaconErrorMessage = ({ message, setErrorAlertClosed }: BeaconErrorMessageProps) => {
```

Comment:
I think a more generalizable parameter format would be to accept an `onClose` function, which is bound directly to the Alert - rather than passing a state setter down. this brings it more in line with antd components, and we could in the future change the onClose behaviour to do additional/different things.

Reply:
yes the state is fine, I mean that the prop on BeaconErrorMessage should be `onClose` and that should be bound to the state update function where this component is used